src/epoch/ut1.rs

```diff
@@ -88,18 +89,19 @@ impl Ut1Provider {
 
     /// Build a UT1 provider by downloading the data from <https://eop2-external.jpl.nasa.gov/eop2/latest_eop2.long> (long time scale UT1 data) and parsing it.
     pub fn download_from_jpl(version: &str) -> Result<Self, HifitimeError> {
-        match get(format!(
-            "https://eop2-external.jpl.nasa.gov/eop2/{}",
-            version
-        )) {
-            Ok(resp) => {
-                let eop_data = String::from_utf8(resp.bytes().unwrap().to_vec()).unwrap();
-                Self::from_eop_data(eop_data)
-            }
-            Err(e) => Err(HifitimeError::Parse {
-                source: ParsingError::DownloadError {
-                    code: e.status().unwrap_or(StatusCode::SEE_OTHER),
-                },
+        let url = format!("https://eop2-external.jpl.nasa.gov/eop2/{}", version);
+
+        // Download the file
+        match get(url).call() {
+            Ok(resp) => Self::from_eop_data(resp.into_body().read_to_string().expect(
```

Comment:
Although the previous code used an `unwrap`, and `expect` is definitely better than `unwrap`, I think it's best to return a Parse UnknownFormat if the `read_to_string` fails.

Reply:
Definitely, I did that on my last commit.